src/ucp/rndv/rndv_get.c

```diff
@@ -144,8 +150,11 @@ ucp_proto_rndv_get_zcopy_send_func(ucp_request_t *req,
     ucs_assert(iov.count == 1);
     ucp_proto_common_zcopy_adjust_min_frag(req, rpriv->mpriv.min_frag,
                                            iov.length, &iov, 1, &offset);
-    return ucp_proto_rndv_get_common_send(req, lpriv, &iov, offset,
-                                          &req->send.state.uct_comp);
+    status = ucp_proto_rndv_get_common_send(req, lpriv, &iov, offset,
```

Comment:
the send can be unsuccessful (i.e. status == NO_RESOURCE), then no need to change the lane

Reply:
remove 'status' helper variable and just return the result from ucp_proto_rndv_put_common_send